moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -370,6 +381,24 @@ class MoveItConfigData
    */
   std::vector<ROSControlConfig> getROSControllers();
 
+  /**
+   * \brief Used for adding a sensor plugin configuation prameter to the sensor plugin configuration parameter list
+   *
+   * @param parameter name
+   * @param parameter value
+   * @param parameter comment
```

Comment:
remove 'parameter' from each line

also, either add a real description for each parameter or remove the @param's part as it doesn't serve a purpose currently